script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -5,190 +5,190 @@ My body temperature drops.
+Predicting the outcome of the next two minutes, I want to tear out my heart, which is already ready to burst.//Not sure where the 'I want to' is coming from here? 俺は２分後の結末を予測し、破裂しそうな心臓をかき毟る。
 <Page3>
 Corpses swarm toward Arcueid.
 A procession reminiscent of midsummer in the midst of the extreme cold.
 They're like ants gathering sugar under the blazing sun.
 Arcueid will be killed.
-Those corpses will tear her limb from limb, just as I had, and offer her dismembered body.
+Those corpses will tear her limb from limb, just as I did, and offer up her dismembered body.//Weirdness here with the split personality 俺が／おまえが: あの死体どもに手足をもぎとられ、俺が／おまえがしたように、解体されて献上される。
 <Page4>
 All while I watch from a safe distance.
 Just like that―――#Just like that?
 <Page5>
 "―――I can't."
-That thing is a disaster in human form.
+That thing is just a disaster in human form.
 I shouldn't have pursued him.
 It was beyond arrogant of me to think that I could defeat him.
 <Page6>
 Stopping him from killing anyone else?
 Solving the problem without any more casualties?
-How foolish. I should have let that monster go. I shouldn't have confronted it. I should have waited for it to pass by like a typhoon. If I had done that, then the city may have only ended up being burned to the ground.
+How foolish. I should have looked the other way. I shouldn't have confronted that monster. As with a typhoon, there's no option but to let it pass. If I had done that, then the city may have only ended up being burned to the ground.// あの怪物は見逃すしかなかった。台風のように、通り過ぎるのを待つしかなかった。
 <Page7>
-"―――But, I can't."//text overflow, needs rephrasing
-A person who couldn't do anything. Me, who chose to do nothing.
-A person who wasn't even armed. Me, who carried a weapon.
-Common sense and irrationality flail about within my pressurized brain.
-My consciousness turns to my rear, telling me to run. There's nothing I can do to stop that.
-It's no use. What can she do against that thing? That woman. This is more than enough. Anything more would be overkill. That woman. All she has to do is run away. Just turn away. That's what any decent person would do. That woman is going to be killed.//ruby dots <ま|・><っ|・><と|・><う|・><な|・><人|・><間|・>
+"―――But, I can't."//Do not alter this page without playtesting
+She, who could do nothing. Me, who chose to do nothing.//Crushed to fit on a single line/free up space
+She, who wasn't even armed. Me, who carried a weapon.//Crushed to fit on a single line/free up space
+Common sense and irrationality writhe within my pressurized brain.
+My consciousness turns to my rear, telling me to run. There's nothing I can do to stop him.//Is he unable to stop himself? Or stop Vlov? 逃げようと意識が背後を向く。止めようのない自分がいる。
+It's no use after all. What could I possibly do against an opponent like that? That woman. From here on I'm totally out of my depth. That woman. I should just run away. Just look away. Any decent human would. That woman, will be killed.//Ruby <ま|・><っ|・><と|・><う|・><な|・><人|・><間|・>
```

Comment:
If it's possible, maybe try and highlight each instance of 'that woman'. Would be italics or bold if it could be done.

Reply:
I would if I could ;-;